<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>afterlogin</class>
 <widget class="QMainWindow" name="afterlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>732</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>732</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>731</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1321</width>
      <height>731</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(D:/Teme Proiecte/POO PROIECT/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernamelabel">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Javanese Text</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 12px;
    color: #444444;
    border-radius: 5px;
    padding: 8px;
    border: 1px solid #cccccc;
    background-color: #f2f2f2;
}

QToolButton::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #cccccc;
    border-left-style: solid;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
    background-color: #ffcc99;
}

QToolButton::down-arrow {
    image: url(path/to/down-arrow.png);
    width: 12px;
    height: 12px;
}
QToolButton:hover {
    background-color: #cccccc;
}

QToolButton:pressed {
    background-color: #999999;
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QListWidget" name="homeworksWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #f7f7f7;
    border-radius: 15px;
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 14px;
    color: #444444;
}

QLabel {
    color: #777777;
}

QPushButton {
    background-color: #f2d3bd;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #e1b6a1;
}</string>
    </property>
   </widget>
   <widget class="QListWidget" name="detailsWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #E8E5E5;
    border-radius: 15px;
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 14px;
    color: #444444;
}

QLabel {
    color: #777777;
}

QPushButton {
    background-color: #f2d3bd;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #e1b6a1;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffcc99;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #ffbb77;
}</string>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffcc99;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #ffbb77;
}</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Title: </string>
    </property>
   </widget>
   <widget class="QLabel" name="groupLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Group: </string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework ID: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="reviewButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #acbbb3;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #ffbb77;
}</string>
    </property>
    <property name="text">
     <string>Waiting for review</string>
    </property>
   </widget>
   <widget class="QLabel" name="termLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Term:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
